atomics/T1090.001/T1090.001.yaml

```diff
@@ -35,19 +35,17 @@ atomic_tests:
       default: "false"
   dependencies:
   - description: Squid must be installed and running
-    prereq_command: if [ -x "$(command -v squid --version)" ]; then exit 0; else exit 1; fi;
+    prereq_command: if ([ "#{run_squid}" = "false" ] || ([ -x "$(command -v squid --version)" ] && (ps aux | grep "squid -N" | grep -v "grep"))); then exit 0; else exit 1; fi;
     get_prereq_command: |-
-      if [[ "#{run_squid}" == "true" ]]; then sudo DEBIAN_FRONTEND=noninteractive apt-get -y install squid curl || sudo yum install -y squid curl || sudo dnf install -y squid curl || brew install curl squid; sed -i 's/http_access deny all/#http_access deny all/' /etc/squid/squid.conf; echo "running squid";squid -N -d 1 &; fi
-      if [[ "#{run_squid}" == "false" ]]; then echo "Skipping proxy setup"; else echo "Value #{run_squid} for variable "run_squid" is not valid"; fi
+      [ "#{run_squid}" = "true" ] && ([ -x "$(command -v squid --version)" ] || sudo DEBIAN_FRONTEND=noninteractive apt-get -y install squid curl || sudo yum install -y squid curl || sudo dnf install -y squid curl || brew install curl squid) && (squid -N -d 1 &) && exit 0
+      echo "Value #{run_squid} for variable "run_squid" is not valid" && exit 1
   executor:
     command: |
       export #{proxy_scheme}_proxy=#{proxy_server}:#{proxy_port}
-      curl http://#{test_domain}
+      curl #{test_url}
```

Comment:
Replaced domain with the URL so that atomic can be flexible and people can use either http or https